source/_integrations/bosch_alarm.markdown

```diff
@@ -60,6 +60,10 @@ Since the _Mode 2_ automation user has "superuser" privileges, it bypasses the r
 
 If your authentication changes, the integration will automatically prompt you to enter the new credentials.
 
+## Discovery
+
+This integration supports DHCP discovery, so it should be discovered automatically. If thie IP Address of the panel changes when you are using discovery, the integration will update the address automatically.
+
```

Comment:
This is automatically part os the `{% include integrations/config_flow.md %}` snippet if that applies in the integration manifest in the Core codebase.

../Frenck